opencti-platform/opencti-front/src/components/ColorPickerField.jsx

```diff
@@ -62,13 +62,15 @@ const ColorPickerField = (props) => {
     }
   };
 
+  const { value, ...otherProps } = fieldToTextField(props);
+
   return (
     <>
       <MuiTextField
-        {...fieldToTextField(props)}
+        {...otherProps}
+        value={value ?? ''}
```

Comment:
Reason: `value` can be null. But an input should not be initialized with null or undefined value, instead use an empty string. By so, React can properly track changes with the input.